tiledb/sm/enums/query_condition_combination_op.h

```diff
@@ -113,6 +113,20 @@ inline void ensure_qc_combo_op_string_is_valid(
   ensure_qc_combo_op_is_valid(qc_combo_op);
 }
 
+inline QueryConditionCombinationOp negate_qc_combination_op(
```

Comment:
docstring ? I know the previous don't have any, but let's be the example :stuck_out_tongue: